### ISSUE TITLE
To do

### ISSUE THREAD

**Issue by ezraholm50** [ezraholm50/MultiInstaller#2]
- [ ] Htop
- [ ] Vpn
- [ ] Test every component
- [ ] Install ownCloud (just use rc.local, should be easy to setup.)
- [ ] Samba
- [ ] Backups
- [ ] Landscape-common (have a whiptail that shows the system stats)
- [ ] Change country repo options
- [ ] Vnc
- [ ] Firewall rules for every atomic package

**Comment by ezraholm50** [comment]
@enoch85 what prerequisites does the vm have? In order to have it installed with the MultiInstaller when grabbing your rc.local?

**Comment by enoch85** [comment]
Rc.local is not needed anymore. Everything is handeled in owncloud_install_production.sh.

You need a clean Ubuntu 16.04 server with OpenSSH preinstalled.

**Comment by ezraholm50** [comment]
Cool thanks. How about libreoffice? Webmin?

Op do 12 mei 2016 12:07 schreef Daniel Hansson <notifications@github.com>:

> Rc.local is not needed anymore. Everything is handeled in
> owncloud_install_production.sh.
>
> You need a clean Ubuntu 16.04 server with OpenSSH preinstalled.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ezraholm50/MultiInstaller/issues/2#issuecomment-218714701>
>
-- 

Met vriendelijke groet,

E. Holm

**Comment by enoch85** [comment]
No, everything is done inside the install script. :) Though Webmin is failing right now after the upgrade to Ubuntu 16.04. I reported a bug and they are working on it.

**Comment by ezraholm50** [comment]
Great saves a lot of trouble! Could you link me to the issue? I only had an
error with the backup somerset repo... but its gone now. I got 4 working
webmins on 4 16.04 vm's..?

Op vr 13 mei 2016 08:42 schreef Daniel Hansson <notifications@github.com>:

> No, everything is done inside the install script. :) Though Webmin is
> failing right now after the upgrade to Ubuntu 16.04. I reported a bug and
> they are working on it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ezraholm50/MultiInstaller/issues/2#issuecomment-218964128>
>
-- 

Met vriendelijke groet,

E. Holm

**Comment by enoch85** [comment]
It fails on new installations, the old one still works (becuase it's based on .deb), but the hash sum fails in apt-get so there are no updates.

Here you go: https://sourceforge.net/p/webadmin/bugs/4742/

**Comment by ezraholm50** [comment]
Thanks! So meanwhile dpkg to install it... I'll follow the bug. Thanks
again.

Op vr 13 mei 2016 08:50 schreef Daniel Hansson <notifications@github.com>:

> It fails on new installations, the old one still works (becuase it's based
> on .deb), but the hash sum fails in apt-get so there are no updates.
>
> Here you go: https://sourceforge.net/p/webadmin/bugs/4742/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ezraholm50/MultiInstaller/issues/2#issuecomment-218965258>
>
-- 

Met vriendelijke groet,

E. Holm

**Comment by enoch85** [comment]
That's not a good idea, becuase that would lead to users with old Webmin installs as they are not upgraded during apt-get update/upgrade. I'd rather leave it out for now.

**Comment by ezraholm50** [comment]
Yeah i get it, when webmin is accessed you'll be notified that theres an
update and you can do it from there. I get your point though..

Op vr 13 mei 2016 09:02 schreef Daniel Hansson <notifications@github.com>:

> That's not a good idea, becuase that would lead to users with old Webmin
> installs as they are not upgraded during apt-get update/upgrade. I'd rather
> leave it out for now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ezraholm50/MultiInstaller/issues/2#issuecomment-218966929>
>
-- 

Met vriendelijke groet,

E. Holm

**Comment by enoch85** [comment]
Yeah, I thiught of that aswell. But naah :)
